decaflex/contrib/evaluate.sh

```diff
@@ -0,0 +1,12 @@
+FILES=references/dev/*.out
```

Comment:
This script does not check for `.ret` files for the `e*.decaf` test cases, please consider adding this functionality

Reply:
In the next PR, I will add functionality to check *.ret files as well. Thanks for the heads up.